frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/LoginHandler.java

```diff
@@ -428,7 +428,7 @@ private APIGatewayProxyResponseEvent handleValidCredentials(
                 MFAMethodsService.getMfaMethodOrDefaultMfaMethod(retrievedMfaMethods, null, null);
 
         if (userMfaDetail.isMfaRequired()) {
-            if (defaultMfaMethod.isPresent()) {
+            if (defaultMfaMethod.isPresent() && userMfaDetail.mfaMethodVerified()) {
```

Comment:
Is there a test that confirms the impact of this?

Can we pull this out to a separate piece of work as I'm not sure of the impact this will have to the rest of the users' journey please?